### PR TITLE
[GS End User] [W-8545389] - Checklist Section 2 (Create Constituent Records)

### PR DESCRIPTION
**Pull request by e-ferrando**
# Critical Changes

# Changes
Create custom metadata and labels for Create Constituent Records section in Get Started page
# Issues Closed

# Community Ideas Delivered

# New Metadata
Custom metadata
        GetStartedChecklistItem.Connect_a_Contact_to_an_Organization
        GetStartedChecklistItem.Connect_Two_Contacts_with_Relationships
        GetStartedChecklistItem.Create_Contacts_and_Household_Accounts
        GetStartedChecklistSection.CreateConstituentRecords_NPSP

Custom labels
      gseuChecklistCreateConstituentRecordsTitle
      gseuChecklistCreateConstituentRecordsDesc
      gseuChecklistItemCreateContactAndAccountTitle
      gseuChecklistItemCreateContactAndAccountDesc
      gseuChecklistItemCreateContactAndAccountLink
      gseuChecklistItemCreateContactAndAccountBlueButton
      gseuChecklistItemConnectContactsRelationshipsTitle
      gseuChecklistItemConnectContactsRelationshipsDesc
      gseuChecklistItemConnectContactsRelationshipsLink
      gseuChecklistItemConnectContactsRelationshipsBlueButton
      gseuChecklistItemConnectContactOrganizationTitle
      gseuChecklistItemConnectContactOrganizationDesc
      gseuChecklistItemConnectContactOrganizationLink
      gseuChecklistItemConnectContactOrganizationBlueButton

# Deleted Metadata

### CODE REVIEW

File: src/labels/CustomLabels.labels [hunk 1]
```diff
@@ -8432,6 +8432,118 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <protected>true</protected>
+        <shortDescription>Description for Create Constituent Records section in Get Started</shortDescription>
+        <value>People are the heart of any nonprofit. Explore Nonprofit Success Pack features to learn how to manage your constituents and their household, personal, and professional connections.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistCreateConstituentRecordsTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title for Create Constituent Records section in Gest Started</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:8448]: Please fix small typo in "Get".

File: src/labels/CustomLabels.labels [hunk 2]
```diff
@@ -8432,6 +8432,118 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>Create Constituent Records</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemConnectContactOrganizationBlueButton</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Blue button text for Connect Contacts to Organization checklist item</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:8456]: I don't think we want to use colors for accessibility reasons. I would just say: Button label for Connect Contacts to Organization checklist item

File: src/labels/CustomLabels.labels [hunk 3]
```diff
@@ -8432,6 +8432,118 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <protected>true</protected>
+        <shortDescription>Link text for Connect Contacts to Organization checklist item in Get Started</shortDescription>
+        <value>Learn more about Affiliations in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemConnectContactOrganizationTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title for Connect Contact to Organization checklist item in Get Started page</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:8480]: I think you can remove the word "page" from the end of the description to keep it consistent with others.

File: src/labels/CustomLabels.labels [hunk 4]
```diff
@@ -8432,6 +8432,118 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>Connect a Contact to an Organization with Affiliations</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemConnectContactsRelationshipsBlueButton</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Blue button text for Connect Two Contacts with Relationships checklist item</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:8488]: Again, I would avoid the use of the color for accessibility reasons. Please use: Button text for Connect Two Contacts with Relationships checklist item

File: src/labels/CustomLabels.labels [hunk 5]
```diff
@@ -8432,6 +8432,118 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <protected>true</protected>
+        <shortDescription>Title for Connect Two Contacts with Relationships checklist item in Get Started</shortDescription>
+        <value>Connect Two Contacts with Relationships</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCreateContactAndAccountBlueButton</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Blue button text for Create Contacts and Account checklist item in Get Started</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:8520]: Please use: Button text for Create Contacts and Account checklist item in Get Started

File: src/labels/CustomLabels.labels [hunk 6]
```diff
@@ -8432,6 +8432,118 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>Create Contacts</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCreateContactAndAccountDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description for Create Contacts and Account checklist item in Get Started page</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:8528]: Please remove the word "page". I would also remove the word "Create" to make this consistent with other short descs for the section.

File: src/labels/CustomLabels.labels [hunk 7]
```diff
@@ -8432,6 +8432,118 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>People are the most important resource for a nonprofit, so understanding how we use the Contact object to track them in NPSP is key. The Contact object stores constituent data such as name, address, and employer.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCreateContactAndAccountLink</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link text for Contact and Accounts checklist item in Get Started page</shortDescription>
+        <value>Learn more about Contact and Accounts in Trailhead</value>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:8537]: Contacts should be plural.

File: src/labels/CustomLabels.labels [hunk 8]
```diff
@@ -8432,6 +8432,118 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCreateContactAndAccountTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title for Create Contacts and Account checkbox item in Get Started page</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:8544]: Please change to: Title for Contacts and Accounts checklist item in Get Started.

File: src/labels/CustomLabels.labels [hunk 9]
```diff
@@ -8432,6 +8432,118 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Video Duration displayed in the Video Header Component</shortDescription>
         <value>in 2 minutes</value>
     </labels>
+    <labels>
```
Comment (force2b) [inline src/labels/CustomLabels.labels:8435]: Can you place these in the same labels file alphabetically? There are other labels that start with `gseu` so these new labels should fit in with them.

**Review by force2b — CHANGES_REQUESTED** [review]

**Comment by force2b** [comment]
@e-ferrando There are two issues with these branch names that I didn't notice before: 

1. All branches need to start with `feature/`. Neither this branch or the one it merges into start with `feature/`. You'll be unable to merge branches into master unless they start with feature. Even this branch is questionable because none of the standard builds have run. This issue will have to be fixed in order to continue with any of this work.
1. This branch merges into another child branch. Child branches are defined as a branch with that contains a double underscrore. So, in this case `adoption__W-8545389-Checklist-Section-2--Create-Constituent-Records` (which is technically a child of `adoption`) is set to merge into `adoption__W-8528035-Checklist-Section-1-(Welcome)` which is also a child branch of `adoption`. MetaCI (our CI/CD) built system only supports a single level of parent child branches. In other words, this branch can merge into `feature/adoption' (which would be the correct branch name), but merging one child branch into another child branch will create a ton of merge conflict issues. You happen to be escaping these only because the branches are missing the `feature/` prefix.

My recommendation is to get this branch tested and then fix all of the branch names on all of the PR's you're working with making sure the PR is merging into a parent branch or master, but never into another child branch.
CC @dcaballeroSf

File: src/labels/CustomLabels.labels [hunk 10]
```diff
@@ -7264,6 +7264,118 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <shortDescription>Description for Contacts and Account checklist item in Get Started</shortDescription>
+        <value>People are the most important resource for a nonprofit, so understanding how we use the Contact object to track them in NPSP is key. The Contact object stores constituent data such as name, address, and employer.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCreateContactAndAccountLink</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link text for Contacts and Accounts checklist item in Get Started</shortDescription>
+        <value>Learn more about Contact and Accounts in Trailhead</value>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7369]: Contacts should be plural. Please change this to: Learn more about Contacts and Accounts in Trailhead

**Review by smarovich — COMMENTED** [review]
Labels look good!

**Pull request merged**